projects/cashmere-examples/src/lib/pagination-simple/pagination-simple-example.component.html

```diff
@@ -1,5 +1,13 @@
 <hc-form-field> <hc-label>Total number of items:</hc-label> <input hcInput type="number" [formControl]="totalItemsControl" [value]="totalItemsControl.value"/> </hc-form-field>
 <hc-form-field> <hc-label>Page number:</hc-label> <input hcInput type="number" [formControl]="pageNumberControl" [value]="pageNumberControl.value"/> </hc-form-field>
 <hc-form-field> <hc-label>Page size:</hc-label> <input hcInput type="number" [formControl]="pageSizeControl" [value]="pageSizeControl.value"/> </hc-form-field>
+<hc-form-field>
+    <hc-label>Display width:</hc-label>
+    <hc-select [formControl]="widthControl">
+        <option value="lg">Large</option>
+        <option value="md">Medium</option>
+        <option value="sm">Small</option>
+    </hc-select>
+</hc-form-field>
 
-<hc-pagination [(pageNumber)]="pageNumber" [(pageSize)]="pageSize" [length]="totalItemsControl.value"></hc-pagination>
+<hc-pagination [(pageNumber)]="pageNumber" [(pageSize)]="pageSize" [length]="totalItemsControl.value" [displayWidth]="widthControl.value">></hc-pagination>
```

Comment:
I think you might have ended up with one extra `>` at the end of your first `hc-pagination` tag

Reply:
This should be fixed now.